cmd/ndm_daemonset/probe/mountprobe.go

```diff
@@ -97,6 +132,13 @@ func (mp *mountProbe) FillBlockDeviceDetails(blockDevice *blockdevice.BlockDevic
 	mountProbe := newMountProbe(blockDevice.DevPath)
 	basicMountInfo, err := mountProbe.MountIdentifier.DeviceBasicMountInfo(mount.HostMountsFilePath)
 	if err != nil {
+		if err == mount.ErrAttributesNotFound {
+			klog.Infof("no mount point found for %s. clearing mount points if any",
+				blockDevice.DevPath)
+			blockDevice.FSInfo.MountPoint = nil
+			blockDevice.FSInfo.FileSystem = ""
```

Comment:
Only the mount points needs to be cleared. The filesystem can still be present, even if the disk is unmounted.

Reply:
done